### ISSUE TITLE
/etc/systemd/system/docker.service.d/http-proxy.conf "\n" in value

### ISSUE THREAD

**Issue by 53d117460ec63d70** [Mirantis/launchpad_legacy#49]
A literal ```\n``` is being added to the end of each environment variable value and no newline between ```Environment``` lines in ```/etc/systemd/system/docker.service.d/http-proxy.conf```:
```
[Service]
Environment="https_proxy=http://proxy:8080\n"Environment="NO_PROXY='10.0.0.*'\n"
```
Version:
```
$ launchpad version
version: 1.0.0-rc1
commit: c3f0a0b
```

**Comment by kke** [comment]
Yeah this seems to be because golang doesn't output linefeeds for `\n` when inside backticks. Will be fixed shortly.

**Comment by kke** [comment]
Should be fixed in [1.0.1-beta1](https://github.com/Mirantis/launchpad/releases/tag/1.0.1-beta1)

**Comment by 53d117460ec63d70** [comment]
Confirmed fixed in release:
```
$ launchpad version
version: 1.0.1-beta1
commit: 93405ac
```
